cost-model/src/block_cost_limits.rs

```diff
@@ -38,6 +38,7 @@ pub const MAX_BLOCK_UNITS: u64 =
 
 #[cfg(test)]
 static_assertions::const_assert_eq!(MAX_BLOCK_UNITS, 48_000_000);
+pub const MAX_BLOCK_UNITS_SIMD_0207: u64 = 50_000_000;
```

Comment:
this should be behind #[feature("dev-only-context-utils")]` the only external place it's used is a test. though perhaps the test can simply be in this crate

Reply:
I'm planning on using this elsewhere in scheduler for a default configuration option. Would rather keep backport consistent with the original PR than change it here on something so minor.